cli/aws_orbit/models/changeset.py

```diff
@@ -315,6 +316,13 @@ def dump_changeset_to_ssm(env_name: str, changeset: Changeset) -> None:
     ssm.put_parameter(name=manifest_parameter_name, obj=Changeset.Schema().dump(changeset))
```

Comment:
Do we need this anymore, as we are writing the changeset to s3?

Reply:
There are no longer callers of it..I can remove